### DISCUSSION TITLE
Project Adding People

### DISCUSSION THREAD

**Discussion by RealJapaneseGreenTea** [community/community#64811] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm about to jump off this platform and let me VAs and support team move forward on my project for creating a mobile App.

I was told to jump on here and get things ready, however I'm at a loss, I think I added the project successfully but I can't see the people I added, I started adding a few random users to see if I could see there names anywhere, I'm a bit confused on when I know if they got the invite.

Sorry, I'm more so the web dev guy on these projects not so much the go get a github account started type guy.

Thanks

**Comment by RamtinKosari** [comment]
Hi dear @RealJapaneseGreenTea 

I understand that you're feeling a bit lost with setting up your project and inviting team members on a platform, possibly GitHub. I can provide you with some general guidance to help you navigate through the process.

* Confirm project setup: Double-check if you have correctly set up the project on the platform. Make sure you have created the necessary repository or project structure where your team can collaborate.

* Inviting team members: To invite team members, you typically need their usernames or email addresses associated with their accounts on the platform. Look for an "Invite" or "Add Collaborator" option within your project settings or repository settings. Enter the usernames or email addresses of the team members you want to invite and send the invitations.

* Confirmation of invitations: Once you have sent the invitations, the platform should provide some indication of whether the invitations were successfully sent. It could be a confirmation message or a notification. Check your platform's notification system or the "Collaborators" tab within your project settings to see if the invited users appear in the list.

* Communication with team members: After sending the invitations, it's a good idea to inform your team members about the invitation and provide them with any additional instructions they may need to join the project. You can communicate with them via email, messaging platforms, or any other means of communication your team typically uses.

* Team member acceptance: The invited team members will receive notifications or email invitations to join the project. They need to accept the invitation and create their accounts on the platform if they don't have one already. Once they accept, they should be able to access the project and collaborate with you.

**Reply by RealJapaneseGreenTea** [reply]
I appreciate the detailed response, I'll see if I did all this properly!

**Reply by RamtinKosari** [reply]
thank you so much, ill be so glad if you mark this answer as "answered" ❤️

**Reply by RealJapaneseGreenTea** [reply]
Also I'm not sure what is happening between you two down voting your stuff... :/ odd.

**Comment by nftushar** [comment]
It sounds like you're having trouble with setting up a GitHub repository, inviting collaborators, and understanding the collaboration process. Don't worry, I'm here to help guide you through the process step by step:

1. **Create a Repository**:
   - Log in to your GitHub account.
   - Click on the "+" sign in the upper right corner of the page and select "New repository".
   - Fill in the repository name and description.
   - Choose whether to make it public or private (depending on your needs).
   - You can choose to initialize the repository with a README file (this can be helpful to provide some initial information about the project).

2. **Adding Collaborators**:
   - Go to the repository you just created.
   - Click on the "Settings" tab.
   - In the left sidebar, click on "Collaborators".
   - Enter the GitHub usernames or email addresses of the people you want to invite as collaborators.
   - Click on "Add collaborator".
   - The invited users will receive an email notification, and they need to accept the invitation by clicking the link in the email.

3. **Accepting Invitations** (For Collaborators):
   - Collaborators will receive an email invitation. They need to click the link in the email.
   - They'll be taken to the repository page where they can accept the invitation.

4. **Access and Permissions**:
   - Collaborators will have access to the repository based on the permissions you've granted.
   - There are different permission levels: "Read" access allows viewing the repository, "Write" access allows making changes, and "Admin" access includes repository settings.

5. **Cloning and Making Changes**:
   - Collaborators can clone the repository to their local machines using `git clone <repository_url>`.
   - They can make changes, commit them, and then push the changes back to GitHub using `git push`.

6. **Pull Requests and Review**:
   - When collaborators want to contribute changes to the main repository, they typically create a pull request (PR).
   - A PR allows changes to be reviewed before merging into the main branch.
   - Other collaborators or you can review the changes, ask for clarifications, suggest improvements, etc.

7. **Merging Changes**:
   - After a PR is reviewed and approved, it can be merged into the main branch.
   - Click the "Merge" button on the PR page to incorporate the changes.
   - Optionally, you can choose to squash the commits or rebase before merging.

8. **Issues and Discussions**:
   - GitHub provides tools for issue tracking and discussions related to the project.
   - Issues can be used to track tasks, bugs, enhancements, etc.
   - Discussions can be used to have conversations around specific topics.

Remember, it might take a bit of time to familiarize yourself with these concepts, especially if you're new to version control and GitHub. Don't hesitate to ask specific questions if you're stuck at any particular step. GitHub's documentation is also very comprehensive and can provide detailed guidance on various aspects of using GitHub for collaboration.

**Comment by RealJapaneseGreenTea** [comment]
Okay I'll have my VA team come and do everything, I don't understand any of this forum process and it's hurting my brain trying to figure out. 

I tried to create an account a while back but I got suspended then deleted for some reason? I think it's in poor taste to remove people who are trying to learn this platform. But it's not mine they can do whatever they want, again, poor taste.

Hoping I'm able to get around that part of github, so I've been trying as quickly as I can to learn the platform so maybe I can avoid that?

Not sure.

Thanks guys,

**Comment by RealJapaneseGreenTea** [comment]
Anyway, I got someone checking manually

**Comment by nftushar** [comment]
same hear

**Comment by RealJapaneseGreenTea** [comment]
Still having difficulties with my hires from Upwork to see the invite.
